lib/jormungandr/test/bench/Latency.hs

```diff
@@ -179,9 +265,16 @@ main = do
         fmtResult "getNetworkInfo     " t8
 
         pure ()
-  where
-    twoFixtureWallet ctx =
-        (,) <$> fixtureWallet ctx <*> fixtureWallet ctx
+
+meanAvg :: [NominalDiffTime] -> Double
+meanAvg ts = sum (map realToFrac ts) * 1000 / fromIntegral (length ts)
+
+buildResult :: [NominalDiffTime] -> Builder
+buildResult [] = "ERR"
+buildResult ts = build $ fixedF 1 $ meanAvg ts
+
+fmtResult :: String -> [NominalDiffTime] -> IO ()
+fmtResult title ts = fmtLn ("    "+|title|+" - "+|buildResult ts|+" ms")
```

Comment:
Perhaps we could use `Fmt.padRightF` on the title

Reply:
indeed, `padLeftF` did a trick : 
```
281 fmtResult :: String -> [NominalDiffTime] -> IO ()                                                                                                                                         
282 fmtResult title ts =                                                                                                                                                                      
283     let titleExt = title|+" - " :: String                                                                                                                                                 
284         titleF = padLeftF 25 ' ' titleExt                                                                                                                                                 
285     in fmtLn (titleF+|buildResult ts|+" ms")
```